### PR TITLE
Pull request with type [CREATE] for user [anthonylamark] and object type [ConsoleExtension] - titled [Removable Media Security (secRMM) ConfigMgr console extension]

### PR DESCRIPTION
**Pull request by anthonylamark**
Automated pull request from the Endpoint Configuration Manager Community hub.

### CODE REVIEW

**Review by kmyrup — APPROVED** [review]

**Comment by anthonylamark** [comment]
Hi Microsoft.  Could I trouble someone else to review/approve this pull request?  Thank you very much

**Review by silveyy — APPROVED** [review]

**Review by sihlamMSFT — CHANGES_REQUESTED** [review]
Please see my comment above.

File: Communityhub/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "downloadLocation": "https://www.squadratechnologies.com/StaticContent/ProductDownload/secRMM/9.11.0.0/secRMMConfigMgrConsoleExtension_9_11_11_0.cab",
```
[the remark this answers is not in the document]
Comment (anthonylamark) [reply Communityhub/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json:2]: Hi guys,
OK, not a problem.
I have to rework my 'create cab code' a bit to get this working so should have another push ready a little later today.
Have a great day and thanks!

[the remark this answers is not in the document]
Comment (anthonylamark) [reply Communityhub/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json:2]: Hi again,
My MECM is at tech preview 2203.  I am trying to understand when I can put our 'MECM secRMM console extension version number' back in sync with the rest of our product (we have multiple components and I try to keep them all at the same version #).  Is there a specific date where the 'MECM tech preview release number' becomes the current for the user community?  If true, do you know when that date will be when MECM tech preview 2203 is the current Tech Preview?  I will mark this in my calendar if so, then I can re-sync our version numbers.  Hope this is NOT clear as mud! :-)

[the remark this answers is not in the document]
Comment (kmyrup) [reply Communityhub/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json:2]: The problem exists only in customer using ConfigMgr 2111 release.  Any customers on 2203, or 2107, 2103, etc. will not have an issue.  So, to answer your question exhaustively, this issue will be present for any customers using 2111, and if you want to be sure to not have download failures, you'd need to keep it this way until 2111 is no longer supported (approx. April/May 2023 according to https://docs.microsoft.com/en-us/mem/configmgr/core/servers/manage/current-branch-versions-supported?msclkid=710b1a6eb4fa11eca61b830ba0f3097f ).

[the remark this answers is not in the document]
Comment (anthonylamark) [reply Communityhub/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json:2]: Hmmm, OK then, would you guys be cool with just going with our 9.11.11.0 version (i.e. this pull request) and I will contact my customers to let them know about ConfigMgr 21111 issue?  I am too OCD to keep this up until next year :-)

[the remark this answers is not in the document]
Comment (sihlamMSFT) [reply Communityhub/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json:2]: Yes. That is fine with us. I believe there is a workaround for customers who already has the version 9.11.9.0. I will reply here once i know what the workaround is. But feel free to complete the pull request. Thanks Anthony!

**Review by sihlamMSFT — APPROVED** [review]

**Pull request merged**